go/ql/lib/ext/builtin.model.yml

```diff
@@ -4,4 +4,9 @@ extensions:
       extensible: summaryModel
     data:
       - ["", "", False, "append", "", "", "Argument[0].ArrayElement", "ReturnValue.ArrayElement", "value", "manual"]
-      - ["", "", False, "append", "", "", "Argument[1]", "ReturnValue.ArrayElement", "value", "manual"]
\ No newline at end of file
+      - ["", "", False, "append", "", "", "Argument[1].ArrayElement", "ReturnValue.ArrayElement", "value", "manual"]
+      - ["", "", False, "copy", "", "", "Argument[1].ArrayElement", "Argument[0].ArrayElement", "value", "manual"]
+      - ["", "", False, "max", "", "", "Argument[0]", "ReturnValue", "value", "manual"]
```

Comment:
Per https://pkg.go.dev/builtin#max min and max are variadic

Reply:
Oops, I hadn't realised that there could be more than two arguments. Although, despite what those docs say, if I try to implement the model like that then it doesn't work. It seems that the compiler has special-cased these functions, and they just have an invalid signature and aren't considered variadic. Maybe because of lack of generics originally stopping them from being implemented normally. It's a bit annoying, as we don't have a way of saying "any argument" in models-as-data (according to our docs, at least). I guess we could do `0..1000`. I'll experiment.

Reply:
I asked on the Gophers slack and someone pointed me to [this comment](https://github.com/golang/go/issues/59488#issuecomment-1548505279). It turns out that `min` and `max` were only added in Go 1.21, and after some debate they decided to make them built-in functions because it's nicer to write `max(a, b)` than `Math.Max(a, b)` and they seem to provide a fundamental utility (though Go survived without it for a long time).